### PR TITLE
feat(docs): highlight code block

### LINKED ISSUE #829: [Feature]: Docs Highlight the demo code with shiki
### Describe the feature

Docs Highlight the demo code with shiki

image

### Additional information

- [X] I intend to submit a PR for this feature.
- [ ] I have already implemented and/or tested this feature.

### PR DESCRIPTION
**Pull request by zhangmo8**
### 🔗 Linked issue

close #829 

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

highlight the demo code block

### 📸 Screenshots (if appropriate)

before: 
image

after:
image

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by sadeghbarati** [comment]
https://github.com/unovue/shadcn-vue/issues/372

**Comment by zhangmo8** [comment]
shadcn has code theme now @sadeghbarati 

Do I need to change it to the same theme as him?

**Comment by sadeghbarati** [comment]
Ahh, yes You are right, please go with `github-dark-default` light-mode for code section is a bit odd

**Comment by zhangmo8** [comment]
Hi @sadeghbarati I looked up the mdx themes used by shadcn, and I thought if we could add a code theme option to the header so that users could choose their favorite code theme


Now it's 

image

**Comment by zhangmo8** [comment]
Maybe it will be like this. What do you think? @sadeghbarati 

image

**Comment by sadeghbarati** [comment]
@zhangmo8 It's nice to have this, but I think one code `theme` is enough
I'm in your branch right now and using `shiki/themes/github-dark-default.mjs`

image

**Comment by zhangmo8** [comment]
> @zhangmo8 It's nice to have this, but I think one code `theme` is enough I'm in your branch right now and using `shiki/themes/github-dark-default.mjs`
> 

Alright, I've changed the theme to github-xx-default. If needed, I can submit the code for the theme switching functionality later.

File: apps/www/.vitepress/theme/config/shiki.ts [hunk 1]
```diff
@@ -1,6 +1,41 @@
...
+import { createHighlighterCore } from 'shiki/core'
+import { createJavaScriptRegexEngine } from 'shiki/engine/javascript'
 
-export const cssVariables = createCssVariablesTheme({
-  variablePrefix: '--shiki-',
-  variableDefaults: {},
+export const shikiThemes: ThemeOptions = {
+  light: 'vitesse-light',
+  dark: 'vitesse-dark',
+}
```
Comment (sadeghbarati) [inline apps/www/.vitepress/theme/config/shiki.ts]:
```suggestion
export const shikiThemes: ThemeOptions = {
  light: 'github-light-default',
  dark: 'github-dark-default',
}
```

Comment (zhangmo8) [reply apps/www/.vitepress/theme/config/shiki.ts]: oops, I forgot here, I'm sorry

**Review by sadeghbarati — APPROVED** [review]
Thanks :pray:

**Comment by sadeghbarati** [comment]
@zhangmo8 Could you please check BlockContainer.vue seems to be using the previous shiki way `cssVariables`

**Comment by zhangmo8** [comment]
It seems that I have missed something. Sorry for any inconvenience caused. I will submit another pull request to fix it. @sadeghbarati

**Pull request merged**